typescript/organisations.d.ts

```diff
@@ -37,6 +37,7 @@ export type NewOrganisation = {
   requireTeamMemberMfa?: boolean
   environmentOrdering?: number[]
   formsBuilderAISystemConfigurationIdOverride?: number
+  aiFormsBuilderEnvironmentIds?: number[]
```

Comment:
Thoughts on storing this on the disclaimer? It should prob be permissioned along with that, and not part of data retention which is what this data is permissioned on.

Reply:
Not super keen but I can live it it

Reply:
I don't like moving this to the disclaimer, because the disclaimer is the person that said it was OK to use.  But numerous people can come along later and change which environments that are allowed without a disclaimer (or so I understand). If we put them together the data will be misleading.

Reply:
Great idea.

Reply:
We already do this (Option 1) with some stuff on the org don't we? I know we do on apps.